fdbrpc/LoadBalance.actor.h

```diff
@@ -171,11 +185,24 @@ void addLaggingRequest(Future<Optional<Reply>> reply, Promise<Void> requestFinis
 	}
 }
 
+ACTOR template <class Interface, class Request, class Multi>
+Future< REPLY_TYPE(Request) > loadBalance(
+	Reference<MultiInterface<Multi>> alternatives,
+	RequestStream<Request> Interface::* channel,
+	Request request = Request(),
+	TaskPriority taskID = TaskPriority::DefaultPromiseEndpoint,
+	bool atMostOnce = false, // if true, throws request_maybe_delivered() instead of retrying automatically
+	QueueModel* model = nullptr)
+{
+	TrackedReply<Request> trackedReply = wait(trackedLoadBalance(alternatives, channel, request, taskID, atMostOnce, model));
```

Comment:
This is pretty useful to have, as I've added TraceEvents so many times to figure this out. If I'm not mistaken though, we are not doing anything anywhere with this information now (in context of the the PR), right?

Reply:
This information is being used, to populated the `storageContacted` field in `EventGetKey`, `EventGetValue`, and `EventGetSubRange`, as well as the `proxyContacted` field of `EventContactedProxy`.